src/external/rawspeed/rawspeed-identify.cpp

```diff
@@ -119,7 +119,11 @@ int main(int argc, const char* argv[])
 
     fprintf(stderr, "Loading file: \"%s\"\n", argv[1]);
 
+    #if defined (_WIN32)
```

Comment:
To me _WIN32 should be fine with MS compiler and MingW compiler. For WIN32 we never now, it may or may not be present. Now I don't have enough experience with Microsoft compiler if WIN32 is defined every time _WIN32 is defined. To be checked with MingW compiler too. This is Windows World Roman :)